ksqldb-cli/src/test/java/io/confluent/ksql/cli/CliTest.java

```diff
@@ -1214,33 +1213,6 @@ public void shouldSubstituteVariablesOnRunCommand()  {
         containsString("Created query with ID CSAS_SHOULDRUNCOMMAND"));
   }
 
-  @Test
-  public void shouldPrintStatementInError() throws Exception {
```

Comment:
Should we update this to be test like `shouldNotPrintStatementInError`?

Reply:
Ah, sorry for the confusion. I had added this test in the part 1 PR because of the hacky way I tried to preserve the error message in that PR. This PR reverted all the code that made that necessary as well as the tests for it.